### ISSUE TITLE
A way to bump a group of repositories with a webhook

### ISSUE THREAD

**Issue by skalero01** [dependabot/feedback#715]
Hi, right now my packages are not being updated via the live updates, not sure if there is a normal delay. I'd like to know if there is any way to bump automatically a group of repositories.

I have a package on packagist and have a bunches of repositories that use this package. Can i set a webhook on github for the package repository so this webhook could make dependabot to bump some repositories?

I think that could be a good idea :)

**Comment by rebelagentm** [comment]
👋 Hi!  I think our queue may have been temporarily backed up.  Are you seeing updates normally again?

As for bumping a group of repositories, I'm not sure if that can be done.  @feelepxyz, is the user's requested functionality something that can be done currently?

**Comment by feelepxyz** [comment]
Yes this was probably due to our massive backlog last week, we increased the numbers of workers quite significantly so should be getting through updates faster in future.

@skalero01 are you still seeing this delay?

There's currently no way to bump a group of repositories using a webhook but we have an endpoint you can hit to tell us about a private dependency release: https://github.com/dependabot/api-docs#notify-dependabot-of-a-private-dependency-release
